src/applications/detection/main.cpp

```diff
@@ -481,10 +518,26 @@ int main(int argc, char** argv)
           int pipeId = (fileInFolder.first & 1);
           boost::ptr_list<CCTag> markers;
 #ifdef PRINT_TO_CERR
-          detection(fileInFolder.first, pipeId, imgGray, params, bank, markers, std::cerr, fileInFolder.second.stem().string());
+          detection(fileInFolder.first, pipeId, imgGray, params, bank, markers, std::cerr, inputImageFile.stem().string());
 #else
           detection(fileInFolder.first, pipeId, imgGray, params, bank, markers, outputFile, fileInFolder.second.stem().string());
```

Comment:
Use inputImageFile in the else case as well?

Reply:
yes, bad refactoring...

Reply:
9c817cd